src/pass_manager.cpp

```diff
@@ -82,6 +84,8 @@ module& get_module(module_pass_manager& mpm) { return mpm.get_module(); }
 
 void run_passes(module& mod, const std::vector<pass>& passes, tracer trace)
 {
+    if(enabled(MIGRAPHX_TRACE_PASSES{}))
```

Comment:
Don't we already have `MIGRAPHX_TRACE_COMPILE` flag that would instantiate `tracer` with `std::cout` ?

Reply:
I think you want to dump pass output with `run_passes` without having to call `compile`.  Is that why is it ?

Reply:
The unit tests dont run compile.